packages/ui-text/src/Text/types.ts

```diff
@@ -36,14 +43,17 @@ export type TextProps = {
     | 'alert'
     | 'primary-inverse'
     | 'secondary-inverse'
-  elementRef?: (...args: any[]) => any
+  /**
+   * Provides a reference to the underlying html element
+   */
+  elementRef?: (element: HTMLElement | null) => void
   fontStyle?: 'italic' | 'normal'
-  letterSpacing?: 'normal' | 'condensed' | 'expanded'
+  letterSpacing: 'normal' | 'condensed' | 'expanded'
   lineHeight?: 'default' | 'fit' | 'condensed' | 'double'
-  size?: 'x-small' | 'small' | 'medium' | 'large' | 'x-large' | 'xx-large'
+  size: 'x-small' | 'small' | 'medium' | 'large' | 'x-large' | 'xx-large'
   transform?: 'none' | 'capitalize' | 'uppercase' | 'lowercase'
   weight?: 'normal' | 'light' | 'bold'
-  wrap?: 'normal' | 'break-word'
+  wrap: 'normal' | 'break-word'
```

Comment:
I don't think these are required props. (`letterSpacing, size, wrap`)
If you check out the README it shows that the `Text` component doesn't require any props in order to be working.
Also they have defaultProps default values set.

Reply:
Nice catch, thanks